crates/rethnet_rpc_server/src/lib.rs

```diff
@@ -458,13 +539,39 @@ async fn router(state: StateType) -> Router {
 /// accepts an address to listen on (which could be a wildcard like 0.0.0.0:0), and a set of
 /// initial accounts to initialize the state.
 /// returns the address that the server is listening on (with any input wildcards resolved).
-pub async fn run(
+pub async fn serve(
     address: SocketAddr,
     config: RpcHardhatNetworkConfig,
-    genesis_accounts: HashMap<Address, AccountInfo>,
-) -> Result<SocketAddr, std::io::Error> {
+    genesis_accounts: Option<HashMap<Address, AccountInfo>>,
+) -> Result<
+    axum::Server<hyper::server::conn::AddrIncoming, axum::routing::IntoMakeService<Router>>,
+    std::io::Error,
+> {
+    let genesis_accounts = genesis_accounts.unwrap_or(TEST_ACCOUNTS.iter().fold(
+        HashMap::default(),
```

Comment:
What made you decide to default to `TEST_ACCOUNTS` if the user doesn't specify `genesis_accounts`?